### PR TITLE
Add Appletun a la Mode

### PR DESCRIPTION
**Pull request by WarriorGallade**

### CODE REVIEW

File: data/mods/gen9ssb/abilities.ts [hunk 1]
```diff
@@ -78,6 +78,21 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+		name: "Served Cold",
+		onTryHit(target, source, move) {
+			if (target !== source && move.type === 'Ice') {
+				if (!this.boost({def: 2})) {
+					this.add('-immune', target, '[from] ability: Served Cold');
+				}
+				return null;
+			}
+		},
+		flags: {},
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/abilities.ts:93]:
```suggestion
		flags: {breakable: 1},
```

File: data/mods/gen9ssb/moves.ts [hunk 2]
```diff
@@ -145,6 +145,64 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+						if (pokemon.boosts[statPlus] < 6 && statPlus !== randomStat) {
+							stats.push(statPlus);
+						}
+					}
+					randomStat = stats.length ? this.sample(stats) : undefined;
+					if (randomStat) boost[randomStat] = 1;
+					this.boost(boost, pokemon, pokemon);
+				},
+			},
+		],
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:201]: again, some things just dont need secondaries. Status moves  don't typically have secondary effects anyways (see: take heart)

File: data/mods/gen9ssb/moves.ts [hunk 3]
```diff
@@ -145,6 +145,55 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+			pokemon.setItem(item);
+			let stats: BoostID[] = [];
+			const boost: SparseBoostsTable = {};
+			let statPlus: BoostID;
+			for (statPlus in pokemon.boosts) {
+				if (statPlus === 'accuracy' || statPlus === 'evasion') continue;
+				if (pokemon.boosts[statPlus] < 6) {
+					stats.push(statPlus);
+				}
+			}
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:179]: dont immediately return here, just make the recycle happen if the pokemon doesnt have an item

Comment (WarriorGallade) [reply data/mods/gen9ssb/moves.ts:179]: noted, fixing in next commit

**Pull request merged**